eng/Xunit3/Microsoft.Testing.Platform.targets

```diff
@@ -78,6 +78,14 @@
       <_ResultsFileToDisplay Condition="!Exists('$(_ResultsFileToDisplay)')">%(TestToRun.ResultsStdOutPath)</_ResultsFileToDisplay>
     </PropertyGroup>
 
+    <!-- Generate a test report -->
+    <Exec Command="pwsh -command &quot;$(RepoRoot)eng/scripts/gha-testreport.ps1 -TestResultsFolder '$(_TestResultDirectory)' -TestSummaryOutputFolder '$(TestResultsLogDir)'&quot;"
```

Comment:
Why not simply implement stuff using [MTP extensibility](https://learn.microsoft.com/dotnet/core/testing/microsoft-testing-platform-architecture-extensions)?

Reply:
It's an interesting idea, how would you approach it?

Reply:
@RussKie You'd implement an extension as both `IDataConsumer` and `ITestSessionLifetimeHandler`. It will be consume `TestNodeUpdateMessage`s. The test node will have `TimingProperty` which the duration, and will have `TestNodeStateProperty` indicating test status (pass, fail, timeout, skipped, cancelled).

While consuming, you'll keep the data you need stored, then in `ITestSessionLifetimeHandler.OnTestSessionFinishingAsync` you can print the data to OutputDevice (and disable Arcade's redirection), or maybe writing to `GITHUB_STEP_SUMMARY` can be a better idea anyways?